doc/nrf/ug_nrf5340.rst

```diff
@@ -567,6 +567,27 @@ See the following instructions.
    :start-after: fota_upgrades_start
    :end-before: fota_upgrades_end
 
+Simultaneous multi-image DFU
+****************************
+
+The simultaneous update of multiple images is available for testing since |NCS| v1.7.0.
+It allows both the application core and the network core to be updated at the same run.
+
+To enable the simultaneous update of multiple images set the following options:
+
+* :kconfig:`CONFIG_BOOT_UPGRADE_ONLY` - The simultaneous update of multiple images does not support network core image reversion so it is recommended to disable application image reversion.
+* :kconfig:`CONFIG_PCD_APP` - Enable commands exchange with the network core.
+* :kconfig:`CONFIG_UPDATEABLE_IMAGE_NUMBER` - Enable support for multiple updatatable images by setting this option to ``2``.
+
+Additionally the memory partitions for secondary images for both the application and the network core must be defined.
+For the example of such configuration refer to :ref:`ug_thingy53`.
```

Comment:
```suggestion
For an example of such configuration, refer to :ref:`ug_thingy53`.
```
Is there a more specific part of the page to link to? The page is very long, and on a quick skim only the application guide part talks about memory partition.